plugins/lookup/hashi_vault.py

```diff
@@ -116,9 +116,12 @@
       description: Path to certificate to use for authentication.
       aliases: [ cacert ]
     validate_certs:
-      description: Controls verification and validation of SSL certificates, mostly you only want to turn off with self signed ones.
+      description:
+        - Controls verification and validation of SSL certificates, mostly you only want to turn off with self signed ones.
+        - Will be populated with the inverse of C(VAULT_SKIP_VERIFY) if that is set and `validate_certs` is not explicitly
+        - provided (added in community.general 1.3.0).
```

Comment:
```suggestion
          provided (added in community.general 1.3.0).
```
This continues the paragraph in the previous list element, and doesn't start a new one.